{{ cookiecutter.package_name|slugify }}/.github/workflows/test.yml

```diff
@@ -9,48 +9,46 @@ on:
 
 jobs:
 
-  test:
-
+  build:
     runs-on: ubuntu-latest
-
-    strategy:
-
-      matrix:
-        python-version: ["{{ cookiecutter.python_version }}"]
-      fail-fast: false
-
     steps:
-
       - name: Checkout
         uses: actions/checkout@v3
-
-      - name: Set up Python {% raw %}${{ matrix.python-version }}{% endraw %}
-        uses: actions/setup-python@v3
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
         with:
-          python-version: {% raw %}${{ matrix.python-version }}{% endraw %}
-
-      - name: Cache Python environment
-        uses: actions/cache@v3
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build dev Docker image
+        uses: docker/build-push-action@v2
         with:
-          path: .venv/
-          key: {% raw %}venv-${{ runner.os }}-py${{ matrix.python-version }}-${{ hashFiles('poetry.lock') }}{% endraw %}
+          context: .
+          target: dev
```

Comment:
Shouldn't we be building `ci` here? The `dev` image is much larger because it contains developer tools that in principle aren't necessary in CI.